assets/js/components/ChangeBadge.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * ChangeBadge component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { calculateChange, numFmt } from '../util';
+
+export default function ChangeBadge( { previousValue, currentValue } ) {
+	const change = calculateChange( previousValue, currentValue );
+	const isNegative = change < 0;
+
+	return (
+		<div
+			className={ classnames( 'googlesitekit-change-badge', {
+				'googlesitekit-change-badge--negative': isNegative,
```

Comment:
Maybe let's add another state for the zero value? If there is no change (`change === 0`), then the badge should have light grey colour instead of green as for positive value. What do you think?